src/argilla/server/search_engine/__init__.py

```diff
@@ -0,0 +1,16 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from .base import *
```

Comment:
Since the `base` defines the `__all__` variable. The imported components are controlled

See https://docs.python.org/3/tutorial/modules.html#importing-from-a-package

Reply:
ah sorry, didn't see the `__all__`